PSOperationsTests/PSOperationsTests.swift

```diff
@@ -973,4 +1006,54 @@ class PSOperationsTests: XCTestCase {
         XCTAssertEqual(1, opQ.operationCount)
         XCTAssertTrue(op.waitCalled)
     }
+    
+    /*
+        In 9.1 (at least) we found that occasionaly OperationQueue would get stuck on an operation
+        The operation would be ready, not finished, not cancelled, and have no dependencies. The queue
+        would have no other operations, but the op still would not execute. We determined a few problems
+        that could cause this issue to occur. This test was used to invoke the problem repeatedly. While we've
+        seen the opCount surpass 100,000 easily we figured 25_000 operations executing one right after the other was
+        a sufficient test and is still probably beyond typical use cases. We wish it could be more concrete, but it is not.
+    */
+    func testOperationQueueNotGettingStuck() {
+        
+        var opCount = 0
+        var requiredToPassCount = 25_000
+        let q = OperationQueue()
+        
+        let exp = expectationWithDescription("requiredToPassCount")
+        
+        func go() {
+            
+            if opCount >= requiredToPassCount {
+                exp.fulfill()
+                return
+            }
+            
+            let blockOp = BlockOperation {
+                (finishBlock: Void -> Void) in
+                finishBlock()
+                go()
```

Comment:
Good question. If you want to build that test and test it on master, go for it. We found that the problem usually occurred around the time an op is added to the queue, not some much when things are in process. I am not really concerned one way or the other. it doesn't take long to run this test.